src/entity/Company.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, CreateDateColumn, Entity, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class Company {
+  @PrimaryColumn({ type: 'text' })
+  id: string;
+
+  @Column({ type: 'text' })
+  name: string;
+
+  @CreateDateColumn()
+  createdAt: Date;
```

Comment:
Just for the sake of it, do we want updatedAt here as well, useful for most entities?

Reply:
We could add it yeah, don't see any harm in it.